docs/integrations-in-rancher/cluster-api/overview.md

```diff
@@ -0,0 +1,200 @@
+
+```bash
+helm install ...
+    # Passing secret name and namespace for additional environment variables
+    --set cluster-api-operator.cluster-api.configSecret.name=<secret-name>
+```
+
+The following is an example of a user-managed secret `cluster-api-operator.cluster-api.configSecret.name=variables` with `CLUSTER_TOPOLOGY`, `EXP_CLUSTER_RESOURCE_SET` and `EXP_MACHINE_POOL` feature flags set and an extra custom variable:
+
+```yaml title="secret.yaml"
+apiVersion: v1
+kind: Secret
+metadata:
+  name: variables
+  namespace: rancher-turtles-system
+type: Opaque
+stringData:
+  CLUSTER_TOPOLOGY: "true"
+  EXP_CLUSTER_RESOURCE_SET: "true"
+  EXP_MACHINE_POOL: "true"
+  CUSTOM_ENV_VAR: "false"
+```
+
+:::info
+For detailed information on the values supported by the chart and their usage, refer to [Helm chart options](https://turtles.docs.rancher.com/docs/reference-guides/rancher-turtles-chart/values)
+:::
+
+#### Install Rancher Turtles without `Cluster API (CAPI) Operator` as a Helm dependency
+
+:::note
+Remember that if you opt for this installation option, you must manage the CAPI Operator installation yourself. You can follow the [CAPI Operator guide](https://turtles.docs.rancher.com/docs/tasks/capi-operator/intro) in the Rancher Turtles documentation for assistance.
+:::
+
+1. Add the `rancher-turtles` chart from the in https://rancher.github.io/turtles Helm repository before installation:
+
+```bash
+helm repo add turtles https://rancher.github.io/turtles
+helm repo update
+```
+
+2. Install the chart into the `rancher-turtles-system` namespace:
+
+```bash
+helm install rancher-turtles turtles/rancher-turtles --version v0.5.0
+    -n rancher-turtles-system
+    --set cluster-api-operator.enabled=false
+    --set cluster-api-operator.cluster-api.enabled=false
+    --create-namespace --wait
+    --dependency-update
+```
+
+The previous commands tell Helm to ignore installing `cluster-api-operator` as a dependency.
+
+3. This operation could take a few minutes and once completed you can review the installed controller listed below:
+- `rancher-turtles-controller`
+
+## Uninstalling Rancher Turtles
+
+:::caution
+When installing Rancher Turtles in your Rancher environment, by default, Rancher Turtles enables the CAPI Operator cleanup. This includes cleaning up CAPI Operator specific webhooks and deployments that otherwise cause issues with Rancher provisioning.
+
+To simplify uninstalling Rancher Turtles (via Rancher Manager or Helm command), the official Rancher Turtles Helm chart includes a `post-delete` hook that that removes the following:
+- Deletes the `mutating-webhook-configuration` and `validating-webhook-configuration` webhooks that are no longer needed.
+- Deletes the CAPI `deployments` that are no longer needed.
+:::
```

Comment:
```suggestion
:::caution

When installing Rancher Turtles in your Rancher environment, by default, Rancher Turtles enables the CAPI Operator cleanup. This includes cleaning up CAPI Operator specific webhooks and deployments that otherwise cause issues with Rancher provisioning.

To simplify uninstalling Rancher Turtles (via Rancher or Helm command), the official Rancher Turtles Helm chart includes a `post-delete` hook that that removes the following:
- Deletes the `mutating-webhook-configuration` and `validating-webhook-configuration` webhooks that are no longer needed.
- Deletes the CAPI `deployments` that are no longer needed.

:::
```

I can't recall if the stance has changed on using the Rancher Manager term, but we should leave it off to avoid possible issues.

Reply:
Went through and removed "Manager" from body text when referring to Rancher.